packages/uswds-utilities/src/styles/rules/background-color.scss

```diff
@@ -9,8 +9,8 @@ output:
   background-color: [value];
 ----------------------------------------
 example:
-  .bg-red-50v {
-    background-color: #be4900; }
+  .bg-red {
+    background-color: #e52207; }
```

Comment:
```suggestion
    background-color: #e52207; 
  }
```

Reply:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)